docs/pages/linking/overview.mdx

```diff
@@ -0,0 +1,111 @@
+
+<BoxLink
+  title="Linking into other apps"
+  description="Learn how to handle common and custom URL schemes to link other apps from your app."
+  href="/linking/into-other-apps/"
+  Icon={BookOpen02Icon}
+/>
+
+## Linking to your app from other apps or a web browser
+
+[Deep Links](https://en.wikipedia.org/wiki/Deep_linking) are links to a specific URL based content inside an app or a website.
+
+For example, by clicking an advertisement of your product, your app will open on the user's device and they can view that product's details. This product's link that the user clicked may look like:
+
+```html
+myapp://web-app.com/product
+```
+
+This link is constructed by three parts:
+
+- **Scheme**: The URL scheme that identifies the app that should open the URL (example: `myapp://`). It can also be `https` or `http` for non-standard deep links.
+- **Host**: The domain name of the app that should open the URL (example: `web-app.com`).
+- **Path**: The path to the screen that should be opened (example: `/product`). If the path isn't specified, the user is taken to the home screen of the app.
+
+<BoxLink
+  title="Linking to your app"
+  description="Learn how to configure custom URL schemes to create a deep link of your app."
+  href="/linking/into-your-app/"
+  Icon={BookOpen02Icon}
+/>
+
+## Universal linking
+
+Both Android and iOS implement their own systems for routing web URL's to an app if the app is installed. On Android, this system is called App Links, and on iOS it is called Universal Links.
+
+### Android App Links
+
+Android App Links are different from [standard deep links](#into-your-app) as they use regular HTTP and HTTPS schemes and are exclusive to Android devices.
+
+This link type allows your app to always open when a user clicks the link instead of choosing between the browser or another handler from a dialog displayed on the device. If the user doesn't have your app installed, the link takes them to your app's associated website.
+
+<BoxLink
+  title="Configure Android App Links"
+  description={
+    <>
+      Learn how to configure <CODE>intentFilters</CODE> and set up two-way association from a
+      standard web URL.
+    </>
+  }
+  href="/linking/android-app-links/"
+  Icon={AndroidIcon}
+/>
+
+### iOS Universal Links
+
+iOS Universal Links are different from [standard deep links](#into-your-app) as they use regular HTTP and HTTPS schemes and are exclusive to iOS devices.
+
+This link type allows your app to always open when a user clicks the link instead of choosing between the browser or another handler from a dialog displayed on the device. If the user doesn't have your app installed, the link takes them to your app's associated website. You can further configure the website by displaying a banner to the user to open your app using [Apple Smart Banner](/linking/ios-universal-links/#apple-smart-banner).
```

Comment:
```suggestion
This link type allows your app to open when a user clicks an HTTP(S) link pointing to your web domain. If the user doesn't have your app installed, the link takes them to your app's associated website. You can further configure the website by displaying a banner for the user to open your app using [Apple Smart Banner](/linking/ios-universal-links/#apple-smart-banner).
```
I don't use iOS as much, but I don't recall iOS prompting for how to handle a link. Usually one app clobbers another if they're sitting on the same scheme, and I thought that an https link would just go to the browser if there's no universal linking app registered

Reply:
Yes you are right. On IOS, they will be directed to the website. I think I had the wrong context here.